lib/views/course_view/components/livenow_section.dart

```diff
@@ -0,0 +1,336 @@
+                  onTap: () {
+                    Navigator.push(
+                      context,
+                      MaterialPageRoute(
+                        //TODO - chat is enabled in live mode
+                        builder: (context) => VideoPlayerPage(
+                          stream: stream,
+                        ),
+                      ),
+                    );
+                  },
+                );
+              },
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Row _buildSectionTitle(String title) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Text(
+          title,
+          style: const TextStyle(
+            fontSize: 22,
+            fontWeight: FontWeight.bold,
+            color: Colors.black,
+          ),
+        ),
+      ],
+    );
+  }
+
+  Widget _buildNoCoursesMessage(BuildContext context, String title) {
+    return Padding(
+      padding: const EdgeInsets.all(10.0),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Text(
+                title,
+                style: const TextStyle(
+                  fontSize: 22,
+                  fontWeight: FontWeight.bold,
+                  color: Colors.black, // Adjust color as per your theme
+                ),
+              ),
+              const Spacer(),
+              // This will push the title to the start of the Row
+            ],
+          ),
+          const SizedBox(height: 20), // Spacing between title and icon
+          const Center(
+            // Center the icon
+            child: Icon(Icons.folder_open, size: 50, color: Colors.grey),
+          ),
+          const SizedBox(height: 8), // Spacing between icon and text
+          Center(
+            child: Text(
+              'No $title found',
+              textAlign: TextAlign.center,
+              style: const TextStyle(
+                fontSize: 18,
+                fontWeight: FontWeight.bold,
+                color: Colors.grey,
+              ),
+            ),
+          ),
+          const SizedBox(height: 12), // Spacing between text and button
+          Center(
+            // Center the button
+            child: ElevatedButton(
+              onPressed: () {
+                /* Action */
+              },
+              child: Text(
+                _buttonText(title),
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  String _buttonText(String title) {
+    switch (title) {
+      case 'My courses':
+        return 'Enroll in a Course';
+      case 'livenow':
+        return 'No courses currently live';
+      case 'Public courses':
+        return 'No public courses found';
+      default:
+        return 'Discover Courses';
+    }
+  }
+
+  List<Course> _defaultCourses() {
```

Comment:
This shouldn´t be here. There is somethiing similar in `Mockdata`
Take a look at the `courses_overview`